### PR TITLE
Refactorsingle movie class component into functional component

### PR DESCRIPTION
**Pull request by bea-ordonez**
We refactored the single movie component from class to functional because we were not using the state , a functional component was the right choice. This PR is showing our understanding of React fundamentals.

### CODE REVIEW

**Review by LeftyLincoln — APPROVED** [review]
As we weren't using state in SingleMovie, I like this refactor to make it a functional component. Looking great.

File: src/components/SingleMovie/SingleMovie.js [hunk 1]
```diff
@@ -1,38 +1,28 @@
...
+      <p>Genre: {singleMovie.genres}</p>
+      <p>Runtime: {singleMovie.runtime} minutes</p>
+    </div>
+  )
 }
 
-
 export default SingleMovie;
 
 SingleMovie.propTypes = {
```
Comment (niksseif) [inline src/components/SingleMovie/SingleMovie.js:24]: Nice work using propTypes!

File: src/components/SingleMovie/SingleMovie.js [hunk 2]
```diff
@@ -1,38 +1,28 @@
...
-            <p>Synopsis: {singleMov.overview}</p>
-            <p>Rating: {singleMov.average_rating} 🍅 </p>
-            <p>Genre: {singleMov.genres}</p>
-            <p>Runtime: {singleMov.runtime} minutes</p>
-            
-            
-          </div>
-        );
-    };
+const SingleMovie = ({singleMovie, showAllMovies}) => {
```
Comment (niksseif) [inline src/components/SingleMovie/SingleMovie.js:5]: Nice work with destructuring your props.

**Review by niksseif — COMMENTED** [review]
This looks good! I left a few notes for you on this PR, but I will also check your API calls and leave some notes there as well.

**Pull request merged**